src/themes/teams-dark/components/Status/statusStyles.ts

```diff
@@ -0,0 +1,63 @@
+import { pxToRem } from '../../../../lib'
```

Comment:
same story here - we shouldn't replicate the same styles, but rather introduce overrides that are specific to dark theme